### PR TITLE
Added cookbook section for adding new channels

### PR DESCRIPTION
**Pull request by martinmitrevski**
### 🔗 Issue Links

_Provide all Jira tickets and/or Github issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

File: docusaurus/docs/iOS/swiftui/swiftui-cookbook/creating-channels.md [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+
+                Text("Create a group")
+                    .font(fonts.bodyBold)
+                    .foregroundColor(Color(colors.text))
+
+                Spacer()
+            }
+            .padding()
+        }
+        .isDetailLink(false)
```
Comment (testableapple) [inline docusaurus/docs/iOS/swiftui/swiftui-cookbook/creating-channels.md:314]: You sure this should be always `false`? 🤔

```suggestion
        .isDetailLink(UIDevice.current.userInterfaceIdiom == .pad)
```

**Review by testableapple — APPROVED** [review]
LGTM 👍

File: docusaurus/docs/iOS/swiftui/swiftui-cookbook/creating-channels.md (hunk 1 above)
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/swiftui-cookbook/creating-channels.md:314]: Yes, false is fine for here.

File: docusaurus/docs/iOS/swiftui/swiftui-cookbook/creating-channels.md [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+}
+```
+
+The code above builds the UI for adding new channels. It consists of a text input view, where you can search for users available on the platform. When you select a user, it creates a channel view to start sending messages and shows the added users above the text input. 
+
+In the trailing part of the text input, there's a button to add additional users to the chat. 
+
+Below the text input view, there is a list that shows the available users to add to a chat.
+
+In the code above, we are also using some helper views, that help us achieve the desired user interface.
```
[the remark this answers is not in the document]
Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/swiftui-cookbook/creating-channels.md:231]: no, actually it's code above, this one is correct.

**Pull request merged**